index.js

```diff
@@ -41,7 +49,7 @@ module.exports = class Cycled extends Array {
 		return this.step(-1);
 	}
 
-	get reversed() {
+	get indefinitelyReversed() {
```

Comment:
This is a getter, while `indefinitely` is not. Make this one like `indefinitely`.